<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>390</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #212c3d;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Ac">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1f618d;
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>430</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 14px;
    height: 60px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LeftBracket">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RightBracket">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Div">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mult">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Minus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Plus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dot">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Equals">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1f618d;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Log">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mod">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Acos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sqrt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Atan">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Asin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Ln">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Graph">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3a4e63;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}

QPushButton:hover {
    background-color: #34495e;
}

QPushButton:pressed {
    background-color: #2c3e50;
}
</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>290</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    height: 60px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>30</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    height: 60px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color: #2c3e50;
    color: #ecf0f1;
}

QMenuBar::item {
    background-color: #2c3e50;
    padding: 6px 10px;
}

QMenuBar::item:selected {
    background-color: #3498db;
}

QMenu {
    background-color: #2c3e50;
    border: 1px solid #1f2b37;
    padding: 4px;
}

QMenu::item {
    padding: 4px 20px;
	color: #ecf0f1; 
}

QMenu::item:selected {
    background-color: #3498db;
    color: #ecf0f1;
}

QMenu::separator {
    height: 1px;
    background-color: #1f2b37;
    margin: 4px 0px;
}

QMenu::indicator {
    width: 14px;
    height: 14px;
}

QMenu::icon {
    padding-left: 5px;
}</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCredit"/>
    <addaction name="actionDeposit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionCredit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
